docs/SWOP-profile-schema.yaml

```diff
@@ -211,37 +253,99 @@ properties  :
                                 items       : 
                                     type        : object
                                     uniqueItems : true
+
+                      # In our practice, we are using labels only for the SPIFFE selector not envvars.
+                      sllabels:
+                          oneOf:
+                            - const: ~
+                            - type: object
```

Comment:
This is old file.    We need to take content from this file 

https://github.hpe.com/hpe/swarm-learning-docs/blob/master/docs/SWOP-profile-schema.yaml